decidim-admin/app/views/decidim/admin/moderations/index.html.erb

```diff
@@ -51,14 +51,14 @@
               <% end %>
               <td class="actions">
 
-                <% if can? :unreport, moderation %>
+                <% if can?(:unreport, moderation) || allowed_to?(:unreport, :moderation) %>
                   <%= icon_link_to "action-undo",
                                    unreport_moderation_path(id: moderation),
                                    t("actions.unreport", scope: "decidim.moderations"),
                                    class: "action-icon--unreport",
                                    method: :put %>
                 <% end %>
-                <% if !moderation.reportable.hidden? && can?(:hide, moderation) %>
+                <% if !moderation.reportable.hidden? && (can?(:hide, moderation) || allowed_to(:hide, :moderation)) %>
```

Comment:
This should have a `?` at the end? `allowed_to(..)` =>`allowed_to?(...)`

Reply:
You're right, why isn't this discovered by the tests? 😕

Reply:
Yeah, I've also asked myself the same :)